src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/LibraryResourceMenu.svelte

```diff
@@ -185,11 +258,18 @@
         </div>
         {#if isLoading}
             <FullPageSpinner />
-        {:else if !resources}
-            <div class="flex flex-grow flex-col items-center justify-items-center overflow-y-scroll">
-                <div class="mb-4 flex-shrink-0 text-sm text-neutral">
-                    {$translate('page.passage.resourcePane.typeToSearch.value')}
-                </div>
+        {:else if resourceGroupings.length === 0}
+            <div class="flex-coloverflow-y-scroll flex flex-grow">
+                {#if visibleSwish}
+                    <button
+                        on:click={openBookChapterVerseMenu}
+                        class="mx-2 flex h-9 items-center justify-center rounded-lg border border-[#EAECF0] p-2 text-sm"
+                        data-app-insights-event-name="library-menu-passage-search-button-clicked"
+                    >
+                        <span class="me-2"><BookIcon /></span>
+                        {$translate('components.search.passageSearch.value')}
+                    </button>
+                {/if}
             </div>
         {:else if filteredResourceCount === 0}
             <NoResourcesFound {searchQuery} />
```

Comment:
It looks like the above logic made this so it's not ever rendered, but I think we still want it for when no resources are found.